src/components/shared/Carousel.tsx

```diff
@@ -79,19 +117,25 @@ function Carousel(props: CarouselProps): JSX.Element {
             {child &&
               <>
                 {child.topText && <h2 id={'body-text'}> {child.topText} </h2>}
-                {child.child}
-                {child.bottomText && <h2 id={'body-text'}> {child.bottomText} </h2>}
                 {child.animationTime &&
                   <span className='time-bar-container'>
                     <div key={`${reloadTime}-${slideIdx}`} className='timebar'>
                       <div className='time' style={{ '--time': child.animationTime + 's' } as CSSProperties} />
                     </div>
-                    <div onClick={() => setReloadTime(Date.now())}>
-                      <Tooltip text='Replay' >
-                        <button className='replay-button' />
-                      </Tooltip>
-                    </div>
+                    <Tooltip text= {isMuted ? 'Unmute' : 'Mute'}>
+                      { isMuted ? <button className='util-button unmute-button' onClick={()=>{setMute(false); storage.removeItem('isMuted');}} />:
+                        <button className='util-button mute-button' onClick={()=>{setMute(true); storage.setItem('isMuted', 'true');}} />}
+                    </Tooltip>
```

Comment:
you can do something like this! and have css for `sound-off-button` and `sound-on-button`
```suggestion
                    <Tooltip text= {isMuted ? 'Unmute' : 'Mute'}>
                      <button className=`sound-${isMuted ? 'off' : 'on'}-button`  onClick={(()=>handleMutedButtonClick())} />}
                    </Tooltip>
```

Reply:
same with other two